lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -378,6 +380,31 @@ def test_render_certificate_only_for_downloadable_status(self):
         self.assertIn("Cannot Find Certificate", response.content)
         self.assertIn("We cannot find a certificate with this URL or ID number.", response.content)
 
+    @ddt.data(True, False)
+    @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_audit_certificate_display(self, eligible_for_certificate):
+        """
+        Ensure that audit-mode certs are not shown in the web view.
+        """
+        # Convert the cert to audit, with the specified eligibility
+        self.cert.mode = 'audit'
+        self.cert.eligible_for_certificate = eligible_for_certificate
+        self.cert.save()
+
+        self._add_course_certificates(count=1, signatory_count=2)
+        test_url = get_certificate_url(
+            user_id=self.user.id,
+            course_id=unicode(self.course.id)
+        )
+        response = self.client.get(test_url)
+
+        if eligible_for_certificate:
+            self.assertIn(str(self.cert.verify_uuid), response.content)
```

Comment:
Are there other important values that we should check for here?  It's an opportunity to ensure the contract is upheld, so would be good to include more fields if it makes sense.